tests/data_files/test-ca-nc.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please add a rule to `tests/data_files/Makefile` to regenerate this file (or an equivalent file with e.g. different randomization and dates).

Reply:
Will do!